docs/reference/oop/file.md

```diff
@@ -0,0 +1,6 @@
+::: synapseclient.models.File
```

Comment:
Not sure whats up with these markdown files/why they are in this PR. I did not create them. Were they deleted from `develop` recently?